### DISCUSSION TITLE
Upgrade to 5.4.6 on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by gerald-brandt** [firefly-iii/firefly-iii#4344] [Firefly III]
I am running Firefly III version 5.2.8.

**Description**
running composer create-project grumpydictator/firefly-iii --no-dev --prefer-dist firefly-iii-updated 5.4.6 gives the following error:

[InvalidArgumentException]                                                                                                 
  Could not find package grumpydictator/firefly-iii with version 5.4.6 in a version installable using your PHP version, PHP  
   extensions and Composer version.       

**Extra info**
I am running @pedrom34 update script.

![image](https://user-images.githubusercontent.com/11981161/106818007-4b761480-663d-11eb-8064-58fa792ca923.png)

--> 

**Bonus points**

x I have read the FAQ at https://bit.ly/FF3-FAQ
x I added a screenshot
- I added log files 
- I was able to replicate the issue on the demo site.

**Comment by elmig** [comment]
Check your php version against requirements.

You need PHP 7.4.
